test-kit/components/image-driver.ts

```diff
@@ -0,0 +1,14 @@
+import {DriverBase} from 'test-drive-react';
+import {Image} from '../../src';
+
+export class ImageDriver extends DriverBase {
+    public static ComponentClass = Image;
+
+    public get nativeElement() {
+        return this.select('NATIVE_IMAGE') as HTMLImageElement;
+    }
+
+    public get style() {
+        return this.nativeElement.style;
+    }
```

Comment:
I suggest having a getter for the `src` attribute (`.source`). Esp. for external tests.